styles/layouts/LatestNews.module.scss

```diff
@@ -0,0 +1,53 @@
+.latestNewsLayout {
+  @apply grid;
+}
+
+.newsItem {
+  @apply grid grid-cols-5 items-center;
+  @apply m-4 p-4;
+  @apply rounded-xl;
+  @apply duration-300 ease-in-out;
+
+  &:hover {
+    @apply bg-hover-1;
+    @apply shadow-lg;
+    @apply transition-[box-shadow,background-color];
+  }
+
+  .left {
+    @apply col-span-4;
+    @apply text-white;
+    @apply grid grid-cols-1;
+
+    .title {
```

Comment:
why is there a `title` class? this would probably be better off being converted to an `h3` or something like that.